src/components/ui/accordion.tsx

```
@@ -0,0 +1,79 @@
+import * as React from "react"
+import { Accordion as AccordionPrimitive } from "radix-ui"
+
+import { cn } from "@/lib/utils"
+import { CaretDownIcon, CaretUpIcon } from "@phosphor-icons/react"
```

ESLint

[error] `@phosphor-icons/react` import should occur before import of `@/lib/utils`

(import/order)